lib/src/signed_video_h26x_common.c

```diff
@@ -875,10 +875,9 @@ check_and_copy_hash_to_hash_list(signed_video_t *self, const uint8_t *hash, size
 }
 
 /*
- * Updates the |linked_hash| buffer with the upcoming hash. The buffer contains 2 slots for hashes.
- * When a new I NALU is encountered in the stream, the hash of the |linked_hash| buffer is updated
- * with the hash of the I NALU. While updating, all the values in the buffer are shifted to the
- * start of the buffer, and the new hash is copied to the last slot of the buffer.
+ * Updates the |linked_hash| buffer with the |hash|. The buffer contains 2 slots for hashes.
+ * When a new NALU is encountered in the stream, the values in the buffer are shifted, and the
```

Comment:
There is nothing in this function that checks if this is a new NALU and then shifts the buffer. That decision is done by the one using this function.
Remove that part from the description.